core/templates/core/base_international.html

```diff
@@ -1,5 +1,6 @@
 {% load static from staticfiles %}
 {% load trans from i18n %}
+{% load social_tags %}
```

Comment:
to make it easier to determine what is being imported here please do something like:

```py
{% from social_tags import build_twitter_link etc %}
```